resources/syntax/Twig.sublime-syntax

```diff
@@ -66,7 +66,7 @@ contexts:
       push: statement_block_meta
 
   statement_block_meta:
-    - meta_scope: meta.statement.twig
+    - meta_scope: meta.template.twig meta.statement.twig
```

Comment:
```suggestion
    - meta_scope: meta.embedded.statement.twig
```

Reply:
Note: Punctuation scopes should be adjusted to `punctuation.section.embedded` accordingly.

Reply:
I appreciate the attention to detail, but I'm trying to make these PRs as succinct as possible, without a lot of extra changes. If you would like to update some of the syntax to use better scopes, you have a lot more knowledge than me and would be better suited to open a PR for that.

Reply:
I'm definitely not happy about the workarounds, but there isn't any alternative at this time. Seeing as currently there is no top level scope available after they are cleared, this is just a small improvement. I have no idea how you would re-add `text.html.twig` since that's all handled in the main syntax file, which is not aware of the HTML/CSS/JS variants.